<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1241</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.153409 rgba(0, 145, 174, 255), stop:1 rgba(184, 50, 50, 255));
border-radius: 10px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_Live">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>511</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Cooper Black&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.210227 rgba(57, 230, 255, 255), stop:1 rgba(165, 0, 198, 255));
border-radius: 10px;
background0color::hover: 
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.0113636 rgba(255, 25, 92, 255), stop:1 rgba(0, 85, 255, 255));</string>
     </property>
     <property name="text">
      <string>Live Covid Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Rapidcovid">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>460</y>
       <width>511</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Cooper Black&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.210227 rgba(57, 230, 255, 255), stop:1 rgba(165, 0, 198, 255));
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Rapid Covid Test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Hospital">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>270</y>
       <width>511</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Cooper Black&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.210227 rgba(57, 230, 255, 255), stop:1 rgba(165, 0, 198, 255));
border-radius: 10px;
background0color::hover: 
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.0113636 rgba(255, 25, 92, 255), stop:1 rgba(0, 85, 255, 255));</string>
     </property>
     <property name="text">
      <string>Hospitals</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/doctor.png</normaloff>../img/doctor.png</iconset>
     </property>
     <property name="shortcut">
      <string>A, I</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_vaccine">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>511</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Cooper Black&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.210227 rgba(57, 230, 255, 255), stop:1 rgba(165, 0, 198, 255));
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Vaccination</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_home">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>1181</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
<!--<<<<<<< HEAD:CoviCURE/Forms/Main/dashboard.ui
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 184, 221, 255), stop:0.971591 rgba(0, 205, 137, 255));</string>
=======-->
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(31, 174, 126, 255), stop:1 rgba(97, 167, 209, 255));
image: url(:/resources/img/CoviCURE.png);</string>

     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Out">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/resources/img/power-button.png</normaloff>:/resources/img/power-button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>85</width>
       <height>85</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
